src/main/resources/conf/yearsDifference.conf

```diff
@@ -0,0 +1,10 @@
+input {
+  years {
+    first = "2016.csv"
```

Comment:
I'd configure that differently - there's start year and end year + file name is later derived based on that configuration?

Reply:
Filenames could be any, it's just in this case they match year's names

Reply:
Discussed internally. Looking for commit.